token/js/src/state/account.ts

```diff
@@ -157,7 +157,14 @@ export async function getMinimumBalanceForRentExemptAccountWithExtensions(
     return await connection.getMinimumBalanceForRentExemption(accountLen, commitment);
 }
 
-function unpackAccount(info: AccountInfo<Buffer> | null, address: PublicKey, programId: PublicKey) {
+/**
+ * Unpacks a token account
+ * @param info the token account on-chain account
+ * @param address TokenAccount
+ * @param programId SPL Token program account
+ * @returns
+ */
+export function unpackAccount(info: AccountInfo<Buffer> | null, address: PublicKey, programId: PublicKey): Account {
```

Comment:
These `programId` args can default to `TOKEN_PROGRAM_ID` since we're exposing them now.

Reply:
Yes, I realized this, added it but forgot to push...